packages/react-components/react-dialog/src/stories/Dialog/DialogNoFocusableElement.stories.tsx

```diff
@@ -15,7 +15,7 @@ export const NoFocusableElement = () => {
           <DialogTitle>Dialog Title</DialogTitle>
           <DialogBody>
             <p>⛔️ A Dialog without focusable elements is not recommended!</p>
-            <p>⛔️ Escape key doesn't work here</p>
+            <p>⚠️ Escape key only works with native dialog</p>
```

Comment:
If I correctly understand the goal: `div` & `dialog` should be replaceable. So it would be great to align behaviors. Otherwise we could get a different behaviors across browsers that might be confusing.

Reply:
The argument here is... do we care for an incompatible behaviour on a very obscure edge case? is it worth adding a global listener that will not follow React's typing API to just resolve that very specific edge case?!

Reply:
another alternative would be to stop support Escape key to work on native dialog also. @layershifter

Reply:
> another alternative would be to stop support Escape key to work on native dialog for this scenario also. @layershifter

@bsunderhus It's also an option 👍 I think that we need to normalize behavior anyway, but it's your choice what to choose 😉